### PR TITLE
Added indexes for the labels

### PR DESCRIPTION
**Pull request by anandjakhaniya**

### CODE REVIEW

File: backend/src/main/resources/liquibase/create-index-changelog.xml [hunk 1]
```diff
@@ -1161,4 +1161,29 @@
             <column name="time_updated"/>
         </createIndex>
     </changeSet>
+
+    <changeSet author="anandJ" id="index_labels_mapping_entity_hash_entity_type">
```
Comment (ravishetye) [inline backend/src/main/resources/liquibase/create-index-changelog.xml]: roll back tag missing

Comment (ravishetye) [reply backend/src/main/resources/liquibase/create-index-changelog.xml]: move this to db-changelog-2.0.xml

**Review by ravishetye — CHANGES_REQUESTED** [review]

File: backend/src/main/resources/liquibase/create-index-changelog.xml (hunk 1 above)
Comment (anandjakhaniya) [reply backend/src/main/resources/liquibase/create-index-changelog.xml]: Moved to db-changelog-2.0.xml with rollback tag

File: backend/src/main/java/ai/verta/modeldb/versioning/RepositoryDAORdbImpl.java [hunk 2]
```diff
@@ -329,8 +329,7 @@ public RepositoryEntity getProtectedRepositoryById(
     return Optional.ofNullable((RepositoryEntity) query.uniqueResult());
   }
 
-  private Optional<RepositoryEntity> getRepositoryByName(
-      Session session, String name) {
+  private Optional<RepositoryEntity> getRepositoryByName(Session session, String name) {
```
Comment (anandjakhaniya) [inline backend/src/main/java/ai/verta/modeldb/versioning/RepositoryDAORdbImpl.java:332]: auto-formatting

**Review by ravishetye — APPROVED** [review]

**Pull request merged**